smoke-test/express/bin/www

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
```

Comment:
This was generated by the express CLI tool. https://expressjs.com/en/starter/generator.html